src/Symfony/Bundle/WebServerBundle/Command/ServerRunCommand.php

```diff
@@ -132,7 +132,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $server = new WebServer();
             $config = new WebServerConfig($documentRoot, $env, $input->getArgument('addressport'), $input->getOption('router'));
 
-            $io->success(sprintf('Server listening on http://%s', $config->getAddress()));
+            $message = sprintf('Server listening on http://%s', $config->getAddress());
+            if ('' !== $extra = $config->getExtraAddress()) {
+                $message = sprintf('%s (http://%s)', $message, $extra);
```

Comment:
To make it a bit easier to understand for users, we could change the output from this:

```
[OK] Server listening on http://0.0.0.0:8080 (http://172.20.0.2:8080)
```

to something like this:

```
[OK] Server listening on http://0.0.0.0:8080 (resolved as http://172.20.0.2:8080)
```

Reply:
@javiereguiluz will do the change as `resolved`, this should also match the method name change (more consistent)